Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -556,11 +556,11 @@ def genS(fragment,howMuch):
                            '--datatier':'GEN-SIM-DIGI-RECO,DQMIO',
                            '--relval':'27000,3000'},
                           step1Defaults])
-step1FastUpg2015Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,L1Reco,RECO,EI,HLT:@relval25ns,VALIDATION:@standardValidation,DQM:@standardDQM',
+step1FastUpg2015Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,L1Reco,RECO,EI,DIGI2RAW,HLT:@relval25ns,VALIDATION:@standardValidation,DQM:@standardDQM',
```

Comment:
OK, let's see what we have from the test. In the corresponding 81x PR https://github.com/cms-sw/cmssw/pull/14098 , i changed it to relval2016 too because I see the corresponding era for these fastsim workflows has been changed to Run2_2016 already.